src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/HttpApi/ApiClient.cs

```diff
@@ -534,12 +535,27 @@ private async Task<ResponseStreamHolder> CaptureMetricsAsync(int processId, int
             throw await CreateUnexpectedStatusCodeExceptionAsync(responseBox.Value).ConfigureAwait(false);
         }
 
-        public async Task<ResponseStreamHolder> CaptureExceptionsAsync(int processId, ExceptionFormat format, CancellationToken token)
+        public async Task<ResponseStreamHolder> CaptureExceptionsAsync(HttpMethod method, ExceptionsConfiguration configuration, int processId, ExceptionFormat format, TimeSpan timeout)
```

Comment:
```suggestion
        public async Task<ResponseStreamHolder> CaptureExceptionsAsync(ExceptionsConfiguration configuration, int processId, ExceptionFormat format, TimeSpan timeout)
```

This is always a POST, no need to add a parameter.